exercises/roman-numerals/description.md

```diff
@@ -10,7 +10,7 @@ This made writing and dating extensive histories of their exploits slightly more
 For example the BBC uses Roman numerals to date their programs.
 
 The Romans wrote numbers using letters - I, V, X, L, C, D, M.
-(notice these letters have lots of straight lines and are hence easy to hack into stone tablets).
+(Notice these letters have lots of straight lines and are hence easy to hack into stone tablets).
```

Comment:
Line 13 appears as an aside from line 12 so I'd remove the period and keep notice lowercase.

"The Romans wrote numbers using letters - I, V, X, L, C, D, M  (notice these letters have lots of straight lines and are hence easy to hack into stone tablets)."

That suggested fix applies to any other asides bracketed by parentheses.

Reply:
This would mean removing the `.` from the end of line 12 and moving it to the end of line 13? And joining the two lines.

Reply:
If it's one sentence, should it be:

```
The Romans wrote numbers using letters - I, V, X, L, C, D, M; notice these letters have lots of straight lines and are hence easy to hack into stone tablets.
```

Reply:
That works too. They're both complete sentences so there's no reason they can't be joined with a semi-colon. As an aside, it's longer than the rest of the sentence so it looks a little weird in my original suggestion.

Reply:
Was your original sentence to keep them as separate sentences? I thought removing the period meant making them one sentence.

Reply:
I made them one sentence with the intention to keep the original second sentence the aside I thought it was. Your version flows better though with the semi-colon and no parentheses.